src/stub_wrappers.cc

```diff
@@ -3,901 +3,839 @@
 // This program is free software: you can redistribute it and/or modify it under
 // the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
 
-#include "blas/device.hh"
+#include "device_internal.hh"
 
 #if ! defined(BLAS_HAVE_CUBLAS) && ! defined(BLAS_HAVE_ROCBLAS) && ! defined(BLAS_HAVE_ONEMKL)
 
 namespace blas {
-namespace device {
+namespace internal {
 
-// =============================================================================
+//==============================================================================
 // Level 1 BLAS - Device Interfaces
 
-// -----------------------------------------------------------------------------
+//------------------------------------------------------------------------------
 // axpy
-// -----------------------------------------------------------------------------
-// saxpy
-void saxpy(
-    blas::Queue& queue,
+//------------------------------------------------------------------------------
+void axpy(
     device_blas_int n,
     float alpha,
-    float *dx, device_blas_int incdx,
-    float *dy, device_blas_int incdy)
+    float const* dx, device_blas_int incdx,
+    float *dy, device_blas_int incdy,
```

Comment:
remove the d in incdx/y

Reply:
same for other instances in this file

Reply:
I didn't change these in the wrappers.cc files yet. That could be a later PR. Here I just made the minimal changes needed to fix `const`. The stub_wrappers.cc file gets deleted later.